src/lib/autocomplete/autocomplete.spec.ts

```diff
@@ -85,22 +87,18 @@ describe('MdAutocomplete', () => {
           .toContain('California', `Expected panel to display when opened programmatically.`);
     });
 
-    it('should close the panel when a click occurs outside it', async(() => {
+    it('should close the panel when blurred', async(() => {
```

Comment:
Can you include a test checking that the panel is open after changing focus to an md-option?